pkgs/applications/misc/octoprint/default.nix

```diff
@@ -235,6 +240,7 @@ let
               #pytestFlagsArray = [ "-W ignore::DeprecationWarning" ];
               disabledTestPaths = oldAttrs.disabledTestPaths ++ [
                 "tests/asgi/test_asgi_servers.py"
+                "tests/test_wsgi_servers.py"
```

Comment:
Why is this safe to ignore?

Reply:
Because it only fails due to the version mix of octoprint.
It won't fail on a normal build, but due to octoprints quirky requirements (due to python2 compatibility), it fails here.

Reply:
Yes, but the same version mix will be used at runtime right? Does this test failure indicate any issues that a user might see during actual usage?

Reply:
at least not for me..